ckanext/push_errors/tests/test_push_errors.py

```diff
@@ -0,0 +1,82 @@
+from unittest import mock
+import pytest
+from ckan.lib.helpers import url_for
+from ckan.tests import factories
+from ckanext.push_errors.blueprints import push_errors
+
+
+class TestPushErrorView:
+    """Tests for the push_errors blueprint"""
+
+    @mock.patch('ckanext.push_errors.blueprints.push_errors.log')
+    def test_unauthorized_no_user(self, mock_log, app):
+        """
+        Test that a 403 response is returned when no user is logged in
+        """
+        with app.flask_app.app_context():
+            with mock.patch.object(push_errors.toolkit, '_', lambda x: x):
+                client = app.test_client()
+                response = client.get('/push-error/test')
```

Comment:
```suggestion
        url = url_for('/push-error/test', msg='test')
        response = app.get(url)
```
Is the context really required?
Next test is not using it